day2_homework/anatomical.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Read the lines into an array of floats
+pixel_values = np.loadtxt('anatomical.txt')
+
+# Find the size of a slice over the third dimension
+P = pixel_values.shape[0]/32.0
+# Make a list of candidates for I
+I = np.arange(120,200)
+mod = P%I
+inds = mod==0
```

Comment:
Nice - this is well 'vectorized' - done in an array in one shot, rather than a loop.